File: backend/app/dto/response/mp_host.go
package response

type MpHost struct {
	ID        uint   `json:"id"`
	Name      string `json:"name"` //主机名
	Host      string `json:"host"` //主机地址
	User      string `json:"user"` //用户名
	IsDefault string `json:"isDefault"`
}

File: frontend/src/api/interface/mp-host.ts
export namespace MpHost {
    export interface HostInfo {
        id: number;
        createdAt: Date;
        name: string;
        host: string;
        user: string;
        pwd: string;
        isDefault: string;
    }

    export interface HostCreate {
        name: string;
        host: string;
        user: string;
        pwd: string;
    }
    export interface HostUpdate {
        id: number;
        name: string;
        host: string;
        user: string;
        pwd: string;
    }
    export interface HostDelete {
        ids: Array<number>;
    }
}
